### PR TITLE
fix up css issues with delete ui

### PR DESCRIPTION
**Pull request by Jonecon**
I must have changed a bunch of css classes but not nuked cache at some point :/

### CODE REVIEW

**Comment by Jonecon** [comment]
Screenshot 2023-11-24 at 11 18 05 AM
Screenshot 2023-11-24 at 11 18 13 AM

File: site/styles/_shift72.scss [hunk 1]
```diff
@@ -278,13 +282,18 @@
 
 s72-useraccount-form {
   .s72-account-password-form-group,
-  .s72-account-pin-form-group {
+  .s72-account-pin-form-group,
+  .s72-account-request-delete-form-group {
```
Comment (jacobtoye) [inline site/styles/_shift72.scss:286]: Is this meant to be in the `s72-useraccount-form` block?

File: site/styles/_shift72.scss [hunk 2]
```diff
@@ -278,13 +282,18 @@
...
     margin-top: 30px;
 
     label {
       font-weight: $font-weight-bold;
     }
   }
+
+  .s72-account-request-delete-message {
+    padding-bottom: 0.5rem;
```
Comment (jacobtoye) [inline site/styles/_shift72.scss:295]: Is `0.5em` a standard spacing amount?

File: site/styles/_shift72.scss (hunk 1 above)
Comment (Jonecon) [reply site/styles/_shift72.scss:286]: Yep, its in the user account form page, the request delete button you see in the first screenshot.

File: site/styles/_shift72.scss (hunk 2 above)
Comment (Jonecon) [reply site/styles/_shift72.scss:295]: Was the amount that was used for the pin text spacing you see in the first screenshot, so I thought I should keep it consistent. Have no idea if that was just thrown in there or not.

Comment (Jonecon) [reply site/styles/_shift72.scss:295]: https://github.com/shift72/core-template/blob/develop/site/styles/_pin-codes.scss#L1

**Review by jacobtoye — APPROVED** [review]

**Pull request merged**